src/OrbitGl/App.h

```diff
@@ -577,6 +577,12 @@ class OrbitApp final : public DataViewFactory,
   void ShowHistogram(const std::vector<uint64_t>* data, const std::string& scope_name,
                      uint64_t scope_id) override;
 
+  // Triggers symbol loading for all modules in ModuleManager that are not loaded yet. This is done
+  // with a simple prioritization. The module `ggpvlk.so` is queued to be loaded first, the "main
+  // module" (binary of the process) is queued to be loaded second. All other modules are queued in
+  // no particular order.
+  void LoadAllSymbols();
```

Comment:
Shouldn't this function return a future that completes when all symbols are loaded (using `orbit_base::WhenAll`)? I know it's not needed for the current use case, but we will need it at the very least when we add tests for this.

Reply:
done